pallets/anmol-nft/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "anmol-nft"
+description = "NFT pallet for extended functionality"
+repository = "https://github.com/open-web3-stack/open-runtime-module-library/tree/master/nft"
+license = "Apache-2.0"
+version = "0.1.0"
+authors = ["Anmol Developers"]
```

Comment:
Please, keep the `authors` consistent with [other packages](https://github.com/anmolnetwork/anmol-node/blob/8e678e0c694b0aa258373ae1e96dc2458737a75a/node/Cargo.toml#L2)

Reply:
Good catch, I corrected the field.